time/src/main/java/net/jqwik/time/internal/properties/configurators/YearMonthRangeConfigurator.java

```diff
@@ -0,0 +1,45 @@
+package net.jqwik.time.internal.properties.configurators;
+
+import java.time.*;
+
+import net.jqwik.api.*;
+import net.jqwik.api.configurators.*;
+import net.jqwik.api.providers.*;
+import net.jqwik.time.api.arbitraries.*;
+import net.jqwik.time.api.constraints.*;
+
+public class YearMonthRangeConfigurator extends ArbitraryConfiguratorBase {
+
+	@Override
+	protected boolean acceptTargetType(TypeUsage targetType) {
+		return targetType.isAssignableFrom(YearMonth.class);
+	}
+
+	public Arbitrary<?> configure(Arbitrary<?> arbitrary, YearMonthRange range) {
+		if (arbitrary instanceof YearMonthArbitrary) {
+			YearMonthArbitrary yearMonthArbitrary = (YearMonthArbitrary) arbitrary;
+			return yearMonthArbitrary.between(isoDateToYearMonth(range.min()), isoDateToYearMonth(range.max()));
+		} else {
+			return arbitrary;
+		}
+	}
+
+	private YearMonth isoDateToYearMonth(String iso) {
+		if (iso == null || iso.length() == 0) {
+			return null;
+		}
+		String[] parts = iso.split("-");
+		if (parts.length != 2) {
+			return null;
```

Comment:
Throw exception when format is wrong.

Reply:
How can I test that this throws an exception?

```
@Example
void dateRangeThrowsException(@ForAll @DateRange(min = "2013-05") Date date){
	assertThatExceptionOfType(DateTimeParseException.class); //not working
}
```

```
timestamp = 2021-01-25T20:05:28.758341700, DateConstraints:dateRangeThrowsException = 
  java.time.format.DateTimeParseException:
    Test
```

Reply:
I've found a solution after experimenting a bit :)

```
@Example
@ExpectFailure(failureType = DateTimeParseException.class)
void dateRangeThrowsException(@ForAll @DateRange(min = "2013-05") Date date){
	//do nothing
}
```